app/models/concerns/hud_reports/clients.rb

```diff
@@ -77,11 +77,16 @@ module HudReports::Clients
       }
     end
 
+    # Assessments are expected if:
+    # You are the head of household.
+    # If the enrollment lasted more than one year
+    # and the reporting period end is more than a year since the beginning of the enrollment
+    # and the enrollment started more than one year ago
     private def annual_assessment_expected?(enrollment)
-      return false if enrollment.last_date_in_program.present? &&
-        enrollment.last_date_in_program - enrollment.first_date_in_program < 365
+      return false unless enrollment.head_of_household?
 
-      enrollment.head_of_household? && enrollment.first_date_in_program + 1.years < report_end_date
+      end_date = [enrollment.last_date_in_program, report_end_date, Date.current].compact.min
```

Comment:
I'm worried about if people run a report with a report end date in the future...

`end_date = [enrollment.last_date_in_program, report_end_date].compact.min || Date.current`?

Reply:
Never mind... That would be pointless... `report_end_date` will never be nil.

Reply:
So, `end_date = [enrollment.last_date_in_program, [report_end_date, Date.current].max].compact.min`>